Material.Icons.WPF/MaterialIcon.cs

```diff
@@ -27,6 +29,14 @@ public MaterialIconKind Kind {
             set => SetValue(KindProperty, value);
         }
 
+        /// <summary>
```

Comment:
I usually combine everything that concerns one property. Move AnimationProperty to Animation

Reply:
What you mean by combine and move?

Reply:
I usually combine everything in groups, like KindProperty, KindPropertyChangedCallback, Kind grouped together. Currently AnimationProperty splits this grouping, so would be nice if AnimationProperty would have been moved to Animation